src/app.js

```diff
@@ -5,10 +5,20 @@ export function app(props, container) {
   var node = toVNode(root, [].map)
   var callbacks = []
   var skipRender
-  var globalState
-  var globalActions
+  var globalState = {}
+  var globalActions = {}
 
-  repaint(flush(init(props, (globalState = {}), (globalActions = {}))))
+  repaint(
+    flush(
+      initModule(
+        props,
+        globalState,
+        globalActions,
+        updateGlobalState,
+        getGlobalState
```

Comment:
saves 3 bytes if we inline `getGlobalState` with
```
function () {
  return globalActions
}
```

Uncompressed
3346 bytes vs 3349

Reply:
Done in the next commit.